app/routes/dashboard.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const router = express.Router();
+const recipesModel = require('../database/models/recipesModel');
+
+/* GET users listing. */
+router.get('/new_recipe', async function(req, res, next){
+    res.render('pages/newrecipe', {title: 'Create Recipe'});
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8445077)
```suggestion
  res.render('pages/newrecipe', {title: 'Create Recipe'});
```